source/portable/os/ota_os_posix.c

```diff
@@ -53,6 +53,11 @@
 #define MAX_MESSAGES      10
 #define MAX_MSG_SIZE      sizeof( OtaEventMsg_t )
 
+/* Time conversion */
+#define OTA_TIME_MS_TO_S( ms )           ( ms / 1000 )
```

Comment:
These macros are local to this file, so prefixing with OTA_TIME isn't necessary.

Reply:
Fixed.